include/itkANTSRegistration.hxx

```diff
@@ -265,28 +265,26 @@ ANTSRegistration<TFixedImage, TMovingImage, TParametersValueType>::GenerateData(
   std::transform(whichTransform.begin(), whichTransform.end(), whichTransform.begin(), tolower);
   typename RegistrationHelperType::XfrmMethod xfrmMethod = m_Helper->StringToXfrmMethod(whichTransform);
 
-  double learningRate = 0.2; // TODO: Make this a parameter
-
   switch (xfrmMethod)
   {
     case RegistrationHelperType::Affine: {
-      m_Helper->AddAffineTransform(learningRate);
```

Comment:
should this be GradientStep or LearningRate?

Reply:
It is called `gradient step` in the documentation. The variable in ANTs\Examples\antsRegistrationTemplateHeader.h was called `learningRate`, and I initially copied the code from there.